kyaml/resid/gvk_test.go

```diff
@@ -47,8 +47,8 @@ var lessThanTests = []struct {
 		Gvk{Group: "a", Version: "c", Kind: "ClusterRole"}},
 	{Gvk{Group: "a", Version: "c", Kind: "Namespace"},
 		Gvk{Group: "a", Version: "b", Kind: "ClusterRole"}},
-	{Gvk{Group: "b", Version: "c", Kind: "Namespace"},
-		Gvk{Group: "a", Version: "d", Kind: "Namespace"}},
+	{Gvk{Group: "a", Version: "d", Kind: "Namespace"},
+		Gvk{Group: "b", Version: "c", Kind: "Namespace"}},
```

Comment:
This is a test case revert from https://github.com/kubernetes-sigs/kustomize/pull/4344/files#diff-148cb3bc0ba93ee6684c49539296bb7045b250be09b3b03cc5073a434319f995L50